File: scripts/quant-score.js
#!/usr/bin/env node
/**
 * Workspace shim → core/quant-score.js
 */
const path = require('path');
const { spawnSync } = require('child_process');

const target = path.join(__dirname, '../core/quant-score.js');
const result = spawnSync(process.execPath, [target, ...process.argv.slice(2)], { stdio: 'inherit' });
process.exit(result.status ?? 1);

File: scripts/quant-score.py
#!/usr/bin/env python3
"""Workspace shim → core/quant_score.py"""
import os
import sys
import subprocess

target = os.path.join(os.path.dirname(__file__), "..", "core", "quant_score.py")
result = subprocess.run([sys.executable, target, *sys.argv[1:]])
sys.exit(result.returncode)